src/modules/feature-modules/accessor/pages/innovation/support/support-update.component.ts

```diff
@@ -201,10 +201,11 @@ export class InnovationSupportUpdateComponent extends CoreComponent implements O
       // this.setAlertSuccess('Support status updated and organisation suggestions sent', { message: 'The Innovation Support status has been successfully updated and the Innovator has been notified of your accompanying suggestions and feedback.' });
       this.setAlertSuccess('Support status updated', { message: 'The innovation support status has been successfully updated.' });
```

Comment:
As you added the redirect alert below, this alert, should be inside the "if", right?

Reply:
Agreed. I'll fix it right now, thanks.